dsc/src/main.rs

```diff
@@ -930,6 +930,7 @@ struct MonitorInfo {
 
 /// State of a downstairs.
 #[derive(Debug, Copy, Clone, Deserialize, Serialize, JsonSchema)]
+#[serde(rename_all = "snake_case")]
```

Comment:
Flagging this for attention, any older client will have its requests rejected after this change.